src/robot2017/src/drive_node.cpp

```diff
@@ -74,9 +115,8 @@ int main(int argc, char **argv)
             pub_fb.publish(motorFb);
         }
 
+        publishStatus();
```

Comment:
not sure where you want it to be set. I figured there would be something in the ros::ok() loop.